.github/workflows/build.yml

```diff
@@ -62,8 +62,7 @@ jobs:
           docker system df -v
       - name: Export docker container
         run: |
-          docker tag "lfedge/eve:$VERSION-kvm" "$TAG-kvm-${{ env.ARCH }}"
-          docker save "$TAG-kvm-${{ env.ARCH }}" > "eve-kvm-${{ env.ARCH }}.tar"
+          make V=1 cache-export IMAGE="lfedge/eve:$VERSION-kvm" OUTFILE="eve-kvm-${{ env.ARCH }}.tar"
```

Comment:
It seems that our expectation here (and few lines below) is to have `$TAG-kvm-${{ env.ARCH }}`, i.e. `evebuild/danger:pr2727-kvm-amd64`. Without this tag we are [failing](https://github.com/lf-edge/eve/runs/8203372105?check_suite_focus=true#step:4:29). As I understand with `docker` flag in `eve` target we will have `lfedge/eve:$VERSION-kvm` in local docker cache. Can we remove modifications and use old code here and few lines below?

Reply:
Yeah, see my longer comment below (I am in the middle of composing it).

Reply:
What do you mean by "old code here"? Just `docker tag` it since we load it anyways? I can try.

Reply:
I mean docker tag and docker save. Without cache-export.

Reply:
Put back and pushed. Let's let it run.

Reply:
And the same logic in that files few lines below

Reply:
CI is green: https://github.com/lf-edge/eve/actions/runs/2999907003, great!

Reply:
Huzzah!